cob_tests/launch/test5_with_nav_static.launch

```diff
@@ -0,0 +1,127 @@
+<launch>   
+    <rosparam command="load" file="$(find cob_tests)/config/test5_with_nav_static.yaml" />
```

Comment:
instead of having two launch files, keep an arg that decides if running a static or dynamic obstacle.